nodejs/__test__/table.test.ts

```diff
@@ -448,6 +448,21 @@ describe("When creating an index", () => {
   });
   afterEach(() => tmpDir.removeCallback());
 
+  it('tests "offset" for "vectorSearch"', async () => {
+    // to trigger an error, change the default limit to 1 on the line below
+    const vectorSearch = (offset: number, limit = 2) =>
```

Comment:
Cannot be merged as-is, because `offset` doesn't work in `vectorSearch`.
When someone decides to fix `offset`, to trigger the error, change default `limit` from 2 to 1.